### PR TITLE
Play -> version 0.1.3

### PR DESCRIPTION
**Pull request by bleehu**
If this is too big, I'll try to split it up.

### CODE REVIEW

**Comment by Turtlelord26** [comment]
FORTH EORLINGAS

**Comment by bleehu** [comment]
Part of the bloat is that we're shipping the repo re-arranging. I don't think we can really reduce that.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
RC -> Play 6/19/18

Changes by File

## Basic Rules

-Wording updates and skill name updates, nonfunctional ✔️
-New penalty for having 1 CHA, functional, untested ❌
---Though, its rattlesnake properties probably have been tested.
-Skill point growth now capped, functional, tested ✔️
-Many rules moved to other subdocuments, nonfunctional ✔️

## Combat Rules

-skill removed from melee attack calculations, functional, tested ✔️


## Damage/Armor Types

-added Electric, Fire, Acid, and Internal damage types, nonfunctional in and of themselves ✔️

## Conditions

-Added Lonely condition to support new 1 CHA penalty, see above

## Glossary

-added IO Ports and Lonely, nonfunctional ✔️

## Species

-File rename, nonfunctional ✔️

## Classes

-Skill renames and list formatting, nonfunctional ✔️
-Added melee weapon proficiencies, functional, untested ❌
---Though, honestly, these are shippable to play because they don't do very much until we rework melee *weapons*
-Added skill proficiencies to class doc, nonfunctional since they're just references to data that was already in the skills doc ✔️
-Priest renamed to Hacker, nonfunctional ✔️
-Adept, Paladin, Monk deleted ✔️

## Feats

-Prerequisites to list format, nonfunctional ✔️
-New feat: Juggler, functional, untested ❌
-Renamed feat: Sleight of Hand -> Muscle Memory, nonfunctional ✔️
-New weapon feats: Melee proficiencies, functional, untested. ❌
---But again, they aren't that big of a deal without reworked melee weaponry.
-Removed monk-specific feat Shadow Sense ✔️
-New feats: Unarmed Combat tree, functional, untested ❌
-Proficiency feat capstones spelled out, nonfunctional ✔️

## Skills
-New skills have supplanted old skills, nonfunctional, tested ✔️
---Note that the Knowledge - General nerf is untested. ❌

## Doctor Procedures
-Needle cost added to Mild Stim. Functional. Testing status unknown ❓✔️

## Engineer Trees
-Gun Emplacement and Snare added to trees, slightly functional, testing not required. ✔️
-{classified} renamed to Tune Power Armor, not evaluatable until we make Power Armor
-Dedicated Process renamed to colocation Processing, nonfunctional because this is the tree ✔️

## Engineer Processes
-Damage typified, functional, untested ❌
-Formatting, nonfunctional ✔
-Combat 2.0 related numbers changes. Some are functional because they changed a bit, but we can't not push them ✔️
-Miscellaneous recourse cost balance tweaks.
-Refractory Spray gains level 10 scaling, functional, untested. ❌
-Cerberus drone, Armor peripheral gain level scaling, functional, untested ❌
-New Peripheral: Flamethrower. Functional, untested ❌
-ASPRO-S now hackable. Functional, untested ❌ ... but then again hacking is still all hacked together pending rework. ❓✔️

## Tome of Spells

-massive deletions ✔️

## Attachments

-Style, nonfunctional ✔️

## Melee/Thrown Weapons

-Size stat added to melee weapons, technically functional ❓
-hand-to-hand attacks replaced with unified Unarmed Strike, functional, untested
-wording changes to support new melee combat maneuvers, nonfunctional ✔️
-removed skill from directly affecting weapon damage and accuracy, functional, tested ✔️


## Pistols, SMG/Shotguns, Armors

-files renamed without changes ✔️

## Carbines, Assault/Battle Rifles

-Missing burstfire rates added, nonfunctional ✔️
-Formatting, nonfunctional ✔️

## Long Rifles

-Style, nonfunctional ✔️

## Heavy Weapons

-Many weapons' magazines increased in storage space requirement, functional, testing unknown ❓

## Items
-New Item, Fishing Pole, testing not required ✔️
-New Item, Universal Translator: functional, untested. ❓✔

## Readme

-Version Bump ✔️
-Directorship change 1source -> turtle ✔️
-1Source and Rose added to dev acknowledgement list, Turtle removed. ✔️

**Comment by bleehu** [comment]
So, the way forward on this would be to revert untested changes? Or do we want to hold for more testing?

**Comment by Turtlelord26** [comment]
Tl;dr, Let's take a couple things out but push most of the ❌ stuff. Reasons below.

Looking at the particular bits that I red-Xed, many fall neatly into buckets

** Melee weapon class starting proficiencies, melee weapon proficiency feats, and melee size weapon stat all proceed from Melee 2.1. I'm not terrifically worried about them, since they don't really apply a balance change themselves but rather shape the balancing environment for an upcoming pass on melee weapons. I can't really feel comfortable in that opinion without someone else's agreement, though, since it's content I made. I will note that our Assassin last weekend obeyed the starting proficiencies and never indicated a problem therewith.

* I do not advise against pushing the melee updates to play.

** The Engineer stuff is almost all untested due to the level of the content, which doesn't make it a lot different from the lv 10+ content already in play. With that in mind, I'd be willing to push the new stuff in as long as we kept Engineer in mind as a possible trouble area (which we should be doing anyway given how much of it existed before the new branch protocols were formalized). The lower-level bits, damage types and the flamethrower, I'm not afraid of. We have good data on the heavy weapon version of the flamethrower, and the damage types really aren't changing the way we were actually handling the processes.

* I advise for pushing the Engineer stuff to play, to keep everything we have to watch for in one place. ✔️

** As for the rest, the UT is hardly going to break anything, but as introducing it involves more vigorously treating different species as having different languages, we may want to exercise discretion here. It's also a standalone component, which makes excluding it for now easier. 

* I advise against pushing the UT to play. ❌

** Heavy weapon magazine size increases may be functional changes but they have a lot of appeal on flavor grounds. I don't think it's a very disruptive change either.

* I advise for pushing heavy magazines to play. ✔️

** As far as the needle cost on mild stim is concerned, that's actually been tested now, courtesy of Zephyr, the Glorious Scale. I don't recall the cost being much of an issue at all.

* I advise strongly that the Mild Stim update be pushed to play. ✔️

** The nerf to Knowledge - General is still untested, but so is the entire skill, really. I think we need to try to work more knowledge checks into our dungeon-building. [Leaves] ... ... ... [Comes back]. I made some mistakes on Tenil's skills list and had to redo it, and one of the decisions that made sense was to take Knowledge - General (I decided that before starting this review). I have now picked the four areas based on his backstory. I found one or two knowledge categories that don't exist and maybe could, but otherwise the process was painless and made me actually think about my character.

* I advise for pushing Knowledge - General to play. ✔️

** Juggler is untested and also a very relevant party may be testing it soon. With that in mind, I think it should stay in R_C for the moment.

* I advise against pushing Juggler to play for now. ❌

** Unarmed combat feats and Unarmed Strike are in the same boat as the Engineer content: hard to test due to the level requirements. I happen to think it's better than the old hand-to-hand, but I shouldn't have the last say.

* I do not advise against pushing unarmed updates

** The new 1 Cha penalty I expect will be fine to ship to play, considering what it's replacing. Also given the repo reorg, it would be rather hard to not ship something in the basic rules docs.

* I advise for pushing Lonely to play. ✔️

**Comment by bleehu** [comment]
I used git-revert to yank the UT, Melee 2.1 and Juggler feat from this promotion. Did I miss anything?

**Comment by Turtlelord26** [comment]
[Sheds single tear for the bits of Melee 2.1]

**Review by Turtlelord26 — APPROVED** [review]
